brainscore_language/models/rnng/test.py

```diff
@@ -0,0 +1,30 @@
+import numpy as np
+import pytest
+
+from brainscore_language import load_model
+from brainscore_language.artificial_subject import ArtificialSubject
+
+
+@pytest.mark.memory_intense
+@pytest.mark.parametrize(
+    "model_identifier, feature_size",
+    [
+        ("rnn-tdg-ptb", 512),
+        ("rnn-lcg-ptb", 512),
+        ("rnn-tdg-ptboanc", 512),
+        ("rnn-lcg-ptboanc", 512),
+        ("rnn-tdg-ptboanc-1024", 1024),
+        ("rnn-lcg-ptboanc-1024", 1024),
+    ],
+)
+def test_neural(model_identifier, feature_size):
```

Comment:
are these models also capable of producing behavioral outputs?

Reply:
in theory yes, but not under the current implementation. There's some non-trivial aspects about how the tasks are currently orchestrated from brainscore, and what the model would need to do to get token-level surprisals. I plan to return to this in the future, but need to set it aside for now for more pressing things.

Reply:
ok sounds good! Will they just raise a `NotImplementedError` when asked to start behavior?

Reply:
Inside the container, a `NotImplementedError` is raised, and then it gets escalated by the `ContainerSubject` class here: https://github.com/brain-score/language/blob/benlipkin/rnng-models/brainscore_language/model_helpers/container.py#L183 which returns the full traceback from inside the container as well as a high-level error message